src/pages/Foundations/Layout/Responsive/code.md

```diff
@@ -107,25 +128,24 @@ $viewports: ('s', 'm', 'l');
 </hintitem>
 
+
 | value      | scss variable      | examples of devices                                                                                                |
 | ---------- | ------------------ | ------------------------------------------------------------------------------------------------------------------ |
-| **0px**    | `$screen-s`        | iphone5, SE                                                                                                        |
-| **320px**  | `$screen-s-mini`   | iphone5, SE                                                                                                        |
-| **360px**  | `$screen-s-medium` | Sony Xperia P, <br>Samsung Galaxy S4, <br>iphone6,7,8                                                              |
-| **390px**  | `$screen-s-mega`   | iphone6+,7+,X+, <br>Samsung Galaxy Note                                                                            |
-| -          | -                  | -                                                                                                                  |
-| **576px**  | `$screen-m`        | tablets                                                                                                            |
-| **680px**  | `$screen-m-mini`   | Microsoft Surface Pro, <br>Apple iPad mini, <br>Samsung Galaxy Tab 7"                                              |
-| **769px**  | `$screen-m-medium` | Samsung Nexus 10, <br>Samsung Galaxy Tab 8.9" & 10"                                                                |
-| **1024px** | `$screen-m-mega`   | Microsoft Surface Pro 3, <br>Apple iPad Pro<br>Ipad (landscape)                                                    |
-| -          | -                  | -                                                                                                                  |
-| **1100px** | `$screen-l`        | landscape tablets, laptops                                                                                         |
-| **1280px** | `$screen-l-mini`   | laptops (depending on settings), <br>Samsung Galaxy Tab 10" (landscape), <br>Samsung Galaxy Tab 2 10" (landscape), |
-| **1440px** | `$screen-l-medium` | laptops (depending on settings), <br>Microsoft Surface Pro 3 (landscape), <br>Apple iPad Pro 12.9" (landscape)     |
-| **1680px** | `$screen-l-mega`   | laptops (depending on settings),                                                                                   |
-| -          | -                  | -                                                                                                                  |
-| **1920px** | `$screen-xl`       | desktop computers                                                                                                  |
-
+| **320px** and up  | `$screen.s-medium`   | iphone5, SE                                                                                                        |
```

Comment:
lacks the 0

Reply:
`$screen.s-medium` => `$screen-s-medium` (replace the dot with a dash everywhere)